### ISSUE TITLE
Link with inline SVG doesn't work

### ISSUE THREAD

**Issue by FelipeMarcos** [luruke/barba.js#59]
Hi,

Link with inline SVG doesn't work; the page is always fully reloaded.

Example:
```html
<a href="http://example.com">
  <svg class="logo" ... >
     <rect ...>
  <svg>
</a>
```

Not related to #58.

I found an article about this:
https://css-tricks.com/links-inline-svg-staying-target-events/

Thanks

**Comment by luruke** [comment]
Hi @FelipeMarcos !
Do you have a working example?
The link is on the same domain?

**Comment by FelipeMarcos** [comment]
Hi @luruke,

[Here is a working example](http://waggish-pin.surge.sh/)
Try pressing the icons - 1/10 the page will be reloaded.

**Comment by luruke** [comment]
Hi @FelipeMarcos ;
It seems working fine to me.

Just two clarification:
- Clicking on a link while an animation is in progress will force the redirect toward that link.
- Clicking on a link wich is the same of the current page will cause a page refresh.

Those two behaviors are normal.

Am I missing something?

**Comment by ndimatteo** [comment]
@luruke weirdly enough I'm getting this same issue. Any inline SVG wrapped in a link is issuing a full reload. Links to the same URL without inline SVG work as expected.

You can take a look here (click the logo + image mask in the center of the screen): http://layout.janicek.nickdimatteo.com

**Comment by luruke** [comment]
Hello @ndimatteo ;
I will fix this in the next version.

Meanwhile a fix could be to add a `pointer-events: none` on `.main-logo svg`.

Cheers

**Comment by pierrehenri220** [comment]
@FelipeMarcos @ndimatteo @luruke 

Greetings, I'm just facing the same situation and looking at this explanation:
- https://stackoverflow.com/questions/12588913/svganimatedstring-missing-method-indexof
It's just an issue with the value of the href attribute returned.

I don't know if you fixed it in the V2 but I used this trick at the moment with the V1:

```
$( '#mysvg' ).on( 'click', 'a', function( event ) {
    event.preventDefault();
    Barba.Dispatcher.trigger('linkClicked', this, { target: this, type: "click" });
    Barba.Pjax.goTo( this.href && this.href.baseVal ? this.href.baseVal : this.href );
});
```

I'm using jQuery and #mysvg is a div containing a bunch of a / svg tags mixed together.
Works with mobile webkit as well. Hope this help.

Cheers
